apps/aehttp/test/aehttp_hyperchains_SUITE.erl

```diff
@@ -265,6 +265,9 @@ end_per_group(_Group, Config) ->
     Config1 = with_saved_keys([nodes], Config),
     [ aecore_suite_utils:stop_node(Node, Config1)
       || {Node, _, _} <- proplists:get_value(nodes, Config1, []) ],
+
+    aecore_suite_utils:assert_no_errors_in_logs(Config1, ["{handled_abort,"]),
```

Comment:
Looks good! I think we only want to ignore the handle_abort in case or parent chain being too short though: `{handled_abort,parent_chain_not_in_sync}` - other `handled_aborts` are not expected!?

Reply:
I pushed https://github.com/aeternity/aeternity/tree/no_errors_in_logs on top of master (since I couldn't be arsed to add your fork), and there are 4 errors, all of them `{handled_abort,parent_chain_not_in_sync}`? What errors do you get in addition?

Reply:
Yes that's the error, but the text reads `{handled_abort,parent_chain_not_synced}` and not `...in_sync`.

Reply:
🙈 just a typo